tailwind.config.js

```diff
@@ -3,7 +3,36 @@ module.exports = {
     './pages/**/*.{js,ts,jsx,tsx}',
     './components/**/*.{js,ts,jsx,tsx}'
   ],
-  theme: {},
+  theme: {
+    colors: {
+      transparent: 'transparent',
+      current: 'currentColor',
+      gray: {
+        0o0: '#ffffff',
```

Comment:
Why 0o0 instead of 00?

Reply:
Hey, thanks for getting back, it is due to error caused due to octal numbers not allowed in strict mode.